frontend/src/components/molecules/GDCPanel.tsx

```diff
@@ -0,0 +1,415 @@
+          Moderate synergy
+        </Text>
+      </Tooltip>
+    );
+  if (corr > 0.1)
+    return (
+      <Tooltip
+        label={`An improvement in the "${name}" KPI would lead to a small improvement in this KPI`}
+      >
+        <Text fontWeight="bold" color="green.600" decoration="underline dotted">
+          Weak synergy
+        </Text>
+      </Tooltip>
+    );
+
+  if (corr <= -0.7)
+    return (
+      <Tooltip
+        label={`An improvement in the "${name}" KPI would lead to an equivalent regression in this KPI`}
+      >
+        <Text fontWeight="bold" color="red.600" decoration="underline dotted">
+          Strong tradeoff
+        </Text>
+      </Tooltip>
+    );
+  if (corr <= -0.4)
+    return (
+      <Tooltip
+        label={`An improvement in the "${name}" KPI would lead to a moderate regression in this KPI`}
+      >
+        <Text fontWeight="bold" color="red.600" decoration="underline dotted">
+          Moderate tradeoff
+        </Text>
+      </Tooltip>
+    );
+  if (corr < 0.1)
+    return (
+      <Tooltip
+        label={`An improvement in the "${name}" KPI would lead to a small regression in this KPI`}
+      >
+        <Text fontWeight="bold" color="red.600" decoration="underline dotted">
+          Weak tradeoff
+        </Text>
+      </Tooltip>
+    );
+
+  return <Text fontWeight="bold">Ambigouos</Text>;
+};
+
+type GDCPanelProps = {
+  year: number;
+
+  municipality: string;
+  data: IndicatorScore | IndicatorWithoutGoal;
+
+  compareMunicipality?: string;
+  compareData?: IndicatorScore | IndicatorWithoutGoal;
+};
+
+const defaultProps = {
+  compareMunicipality: undefined,
+  compareData: undefined,
+};
+
+const GDCView: React.FC<GDCPanelProps> = (props: GDCPanelProps) => {
+  const [isLoadingCorrelated, setLoadingCorrelated] = useState<boolean>(false);
+  const [correlatedKPIs, setCorrelatedKPIs] = useState<CorrelatedKPI[]>();
+
+  const { year, municipality, data, compareMunicipality, compareData } = props;
+
+  const loadCorrelatedKPIs = async () => {
+    setLoadingCorrelated(true);
+
+    const correlations = await getCorrelatedKPIs(data.kpi);
+    setCorrelatedKPIs(correlations.sort((a, b) => b.correlation - a.correlation));
+
+    setLoadingCorrelated(false);
+  };
+
+  const dataIsIndicatorScore = (data as IndicatorScore).goal !== undefined;
+  const compareIsIndicatorScore =
+    compareData !== undefined && (compareData as IndicatorScore).goal !== undefined;
+
+  const dummyGrowth = { value: 0, startYear: year, endYear: year };
+  const bestGrowth =
+    data.yearlyGrowth.length > 0 ? data.yearlyGrowth[data.yearlyGrowth.length - 1] : dummyGrowth;
+  const worstGrowth = data.yearlyGrowth.length > 0 ? data.yearlyGrowth[0] : dummyGrowth;
+
+  const projectedCompletion = dataIsIndicatorScore
+    ? +(data as IndicatorScore).projectedCompletion.toFixed(1)
+    : -1;
+
+  let correlatedTable = null;
+  if (correlatedKPIs !== undefined) {
```

Comment:
There's more context as we're filling table based on the length of the array (with a special case for length 0), so this would be an invalid transformation.